Pirate ship Remake

spockye opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

This is no longer a draft, and im sure i'll be making further changes. if you dont like anything, or do, please tell me
---

This PR simply remakes the Pirate ship!
so, whats changed?
well, the pirates themselves are the same, and the starting equipment is mostly the same, except for some changes:
-1 retro laser gun
+1 doublebarrel shotgun
+misc types of loose shotgun shells + spare ammo
-1 bola
and a lot of misc junk and basic equipment has been added
_______

i wont lie, the old shuttle has a LOT of soul, and i'd totally get it if people dont want to see it go. Though do please give it a chance :)

- [x] make the shuttle fly the right way
- [ ] re-balancing if needed


<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
the previous shuttle was very old, and dated. While i wouldnt say that its BAD, i do think it could use a replacement at this point 
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.
-->

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/ecefb283-5c20-4e38-9803-cf70696557b8)

![StrongDMM-2024-04-03 19 27 38](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/39016f6f-b341-4358-b989-dc72de905b4a)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/412bd2e4-a829-4d8e-8174-019eaa30293e)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/c4bcec7e-1e2c-4672-b59b-762a324966d4)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/2c25d775-d615-477e-901b-c2810ce178c8)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/8541022a-bed5-4cef-8011-274b2ffc2006)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/2d05b3e5-0977-478e-bbbe-cdcaa4c72ff6)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/c7c870d5-77cf-4c7b-bb3e-4670497977ad)

(i'll replace the images once the april event is over. all line decals are red/white)
</details>

## Changelog
:cl:
tweak: Remakes Pirate_default.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

spockye committed:
primer

spockye committed:
Merge remote-tracking branch 'upstream/master' into pirate-ship-base

spockye committed:
misc-floor fixes

spockye committed:
adds a couple drinking glasses

EvilDragonfiend commented on _maps/shuttles/pirate/pirate_default.dmm:
```diff
@@ -1,1357 +1,2564 @@
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0);
+	desc = "Stolen from a cow genetics lab, she can survive in a vacuum safely, her milk is also delicious!";
+	name = "Ol Betsy"
+	},
```
please give pirate faction to cow so turret won't misfire

spockye replied:
fixed, thank you!

spockye committed:
Merge remote-tracking branch 'upstream/master' into pirate-ship-remake

spockye committed:
misc fixes

fixes cow faction
changes some objects to compensate for ship turning

spockye committed:
removes a little more pixel shifting

spockye committed:
hopefully fixes flight direction

spockye committed:
fixes ship flight direction

spockye committed:
responds to review

removes door barricade
moves around and replaces maintenance spawns
moves esword poster
doublestacks fan props
fixes thruster names
adds two extra turrets on the backside
moves the boozeomat
girder wall thingy!
freezer for milkie
adds firedoors
the parrot WILL steal ammo
fixes windoor
fixes backpack locker
replaces the dispensers
removes plasma
replaces laptop with civi version
windows have shutters!
renames the dog bed

spockye committed:
responds to review part 2

adds atmos waste loop

spockye committed:
oops! all firedoors!

fixes firelocks

spockye committed:
fixes engines

spockye committed:
Merge remote-tracking branch 'upstream/master' into pirate-ship-remake

EvilDragonfiend reviewed (approved):
This seems fine enough. We can make another change in the future if it's necessary.
